public/pages/SampleData/components/SampleDataBox/SampleDataBox.tsx

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import React from 'react';
+import ContentPanel from '../../../../components/ContentPanel/ContentPanel';
+import {
+  EuiButton,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiTitle,
+  EuiSpacer,
+  EuiLink,
+} from '@elastic/eui';
+import { PLUGIN_NAME } from '../../../../utils/constants';
+
+interface SampleDataBoxProps {
+  title: string;
+  icon: any;
+  description: string;
+  buttonDescription: string;
```

Comment:
maybe rename it to "LoadDataButtonDescription", "buttonDescription" sounds too generic.

Reply:
sure, will change.